tests/test_templates.py

```diff
@@ -128,8 +127,14 @@ def test_table(self):
     </div>
 </body>
 </html>"""
-
         html = render_response(response=response, status_code=200)
-        expected = [line.strip() for line in expected.splitlines(1)]
-        html = [line.strip() for line in html.splitlines(1)]
+        expected = self._strip_whitespace(expected)
+        html = self._strip_ws(html)
         self.assertEqual(expected, html, "Table response incorrect")
+
+    def _strip_ws(self, in_text):
```

Comment:
My comment probably got lost as there was a push to fix the `def` as I was pushing submit. I think this function is really `_string_to_list()` or something. Filtering out blank lines and stripping trailing whitespace is not the primary purpose.